src/OrbitGl/CaptureWindow.cpp

```diff
@@ -642,13 +470,12 @@ void CaptureWindow::UpdateHorizontalScroll(float ratio) {
 
 void CaptureWindow::UpdateVerticalScroll(float ratio) {
   if (time_graph_ == nullptr) return;
-  float min = world_max_y_;
-  float max = world_height_ - time_graph_->GetTrackManager()->GetTracksTotalHeight();
+  float min = 0.0f;
```

Comment:
`world_max_y_` was 0 before since all coordinates in the world have negative y. This is now managed by the `Viewport`, but maybe it makes sense to retrieve the max-value from there instead of hardcoding it.

Reply:
Yes, or at least a comment to explain why it is 0

Reply:
Done